### PR TITLE
Change Login SDK to support Android Autofill again - 3rd time is a charm...

### PR DESCRIPTION
**Pull request by the-felipeal**
Android Autofill, a new feature released on Android Oreo, was not working
for 3 reasons:

1.WebView was created with the application context.
2.Chrome Custom Tabs do not support Autofill yet.
3.An Android bug that disables autofill when a dialog is resized.

This change fixes these 3 issues by:

1.Passing the activity context to the Webview.
2.Changing the custom tabs logic to disallow it if the device is running
  on Android Oreo and the user enabled Autofill.
3.Working around the Android bug by updating the dialog window params
  with the window params associated with the activity that launched it.

Thanks for proposing a pull request.

To help us review the request, please complete the following:
- [x] sign contributor license agreement: https://developers.facebook.com/opensource/cla
- [x] submit against our `:dev` branch, not `master`.
- [x] describe the change (for example, what happens before the change, and after the change)

### CODE REVIEW

File: facebook-common/src/main/java/com/facebook/login/CustomTabLoginMethodHandler.java [hunk 1]
```diff
@@ -26,9 +26,11 @@
 import android.content.pm.ResolveInfo;
 import android.content.pm.ServiceInfo;
 import android.net.Uri;
+import android.os.Build;
```
Comment (abeatte) [inline facebook-common/src/main/java/com/facebook/login/CustomTabLoginMethodHandler.java]: unneeded

File: facebook-common/src/main/java/com/facebook/login/CustomTabLoginMethodHandler.java [hunk 2]
```diff
@@ -26,9 +26,11 @@
...
 import android.os.Bundle;
 import android.os.Parcel;
 import android.os.Parcelable;
+import android.view.autofill.AutofillManager;
```
Comment (abeatte) [inline facebook-common/src/main/java/com/facebook/login/CustomTabLoginMethodHandler.java]: unneeded

**Comment by abeatte** [comment]
I assume that the toast I see in verifying this fix is presented because of https://github.com/googlesamples/android-AutofillFramework and not something that will be in the UI for all user-facing interactions, correct?

**Comment by abeatte** [comment]
Does this change REQUIRE that our min/target SDK version be 26?

**Comment by the-felipeal** [comment]
The toast is expected because the sample service cannot verify that the app package / signature is "owned" by Facebook; a real service implementation could use a custom UI to ask the user to confirm.

Anyways I forked the service  on https://github.com/the-felipeal/android-AutofillFramework/tree/facebook_login ; this service should work fine.

Regarding the SDK, we only need 26 to compile it (as we're using new APIs), I didn't change the min target version (it's still 25).

Finally, I will try to fix the imports on this PR...

**Comment by abeatte** [comment]
Great, everything looks good here. Thanks for all your hard work. We appreciate it! (Y)

**Comment by carloshwa** [comment]
@abeatte Choosing to disable Chrome Custom Tabs support because it doesn't support autofill seems like a step backwards to me. It effectively disables the ability to sign-in automatically based on my Chrome facebook.com login status on Android O devices. Can we reenable that support?

**Pull request merged**